src/util/GameLogicUtils.cpp

```diff
@@ -80,8 +80,8 @@ namespace spy::util {
             return isPersonOnField(s, currentPoint);
         });
     }
-
-    std::vector<character::Character>::const_iterator GameLogicUtils::findInCharacterSetByCoordinates(const character::CharacterSet &cs, const util::Point &p) {
+  
+  std::vector<character::Character>::const_iterator GameLogicUtils::findInCharacterSetByCoordinates(const character::CharacterSet &cs, const util::Point &p) {
```

Comment:
```suggestion
 character::CharacterSet::const_iterator GameLogicUtils::findInCharacterSetByCoordinates(const character::CharacterSet &cs, const util::Point &p) {
```

Dafür müsste man [hier](https://github.com/SoPra-Team-17/LibCommon/blob/7edd1eb096dc1ffdbc0fedc76911b3624c4dba8e/src/datatypes/character/CharacterSet.hpp#L21) die zwei using statements public machen, dann bindet man das explizit an den iterator der dort verwendet wird und ist nicht davon abhängig die interne Implementierung zu kennen.